src/main/webapp/app/entities/metis/conversation/channel.model.ts

```diff
@@ -3,6 +3,15 @@ import { Exercise } from 'app/entities/exercise.model';
 import { Lecture } from 'app/entities/lecture.model';
 import { Exam } from 'app/entities/exam.model';
 
+// IMPORTANT NOTICE: The following strings have to be consistent with
+// the types defined in ChannelSubType.java, but in lowercase
+// since ChannelDTO.java#getSubType returns a lowercase String
```

Comment:
This special handling seems a bit odd to me, why can we not use the normal enum deserialization?

Reply:
I am using normal deserialization. It is the same behavior as for ConversationType in conversation.model.ts.